src/data/BibleVersionCollection.ts

```diff
@@ -103,4 +103,10 @@ export const BibleVersionCollection: IBibleVersion[] = [
     language: 'Chinese',
     apiSource: BibleAPISourceCollection.bollsLife,
   },
+  {
+    key: 'NR06',
```

Comment:
do you mind following the existing pattern to update the key to lower case?
```suggestion
    key: 'nr06',
```